README.md

```diff
@@ -1,3 +1,32 @@
-# Datadog CloudFoundry Buildpack
+# Datadog Cloud Foundry Buildpack
 
-This buildpack installs Dogstatsd into the container your app is running on.
+This is a [decorator buildpack](https://github.com/cf-platform-eng/meta-buildpack/blob/master/README.md#decorators) for Cloud Foundry. It will install a Datadog Dogstatsd binary on the container your app is running on. It uses the new Agent 6 build of Dogstatsd, which is very light weight and powerful. You should barely notice it running on your container.
```

Comment:
DogStatsD

Reply:
in the container or on the container?

Reply:
I'd probably cut this "It uses the new Agent 6 build of Dogstatsd, which is very light weight and powerful. You should barely notice it running on your container."

Maybe just . "This only contains the DogStatsd components of Datadog Agent, which as limited to no overhead."

Reply:
what's the difference between in and on?